public/gallery/index.html

```diff
@@ -83,6 +83,157 @@
 
 	</head>
 	<body>
+
+    <div id="2017">
+      <header>
+        <a href="/">
+          <img src="brickhack_3.svg">
+        </a>
+        <h1>BrickHack 3</h1>
+        <h2>February 11-12, 2016</h2>
+        <h2>schools - 500 hackers - 24 hours</h2>
```

Comment:
Need a number here

Reply:
Also we should use the all-white version of the logo - this one blends in with the background:

![screenshot 2017-08-28 15 36 25](https://user-images.githubusercontent.com/1441807/29790157-c4c1b368-8c06-11e7-86f1-f289182d9b50.png)